deploy/olm-catalog/nxiq-operator-certified/1.94.0-1/nxiq-operator-certified.v1.94.0-1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,368 @@
+            "persistence": {
+              "accessMode": "ReadWriteOnce",
+              "storageSize": "1Gi"
+            },
+            "service": {
+              "annotations": {},
+              "enabled": false,
+              "labels": {},
+              "ports": [
+                {
+                  "name": "nexus-service",
+                  "port": 80,
+                  "targetPort": 80
+                }
+              ]
+            },
+            "serviceAccount": {
+              "create": true,
+              "name": null
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: "Security"
+    description: |-
+      Nexus Lifecycle is an open source governance platform that continuously identifies risk,
+      automatically enforces policy, and provides visibility throughout the entire SDLC.
+    containerImage: registry.connect.redhat.com/sonatype/nxiq-operator-certified:1.94.0-1
+    repository: https://github.com/sonatype/operator-iq
+    createdAt: 2020-05-01
+    support: Sonatype
+    certified: "true"
+  name: nxiq-operator-certified.v1.94.0-1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: NexusIQ
+      name: nexusiqs.sonatype.com
+      version: v1alpha1
+      description: Nexus IQ Server
+      displayName: NexusIQ
+  description: |-
+    Nexus Lifecycle is an open source governance platform that continuously identifies risk,
+    automatically enforces policy, and provides visibility throughout the entire SDLC.
+    Developers leveraging open source require a solution that helps them make safer choices
+    while still delivering at DevOps speed.
+    Nexus Lifecycle empowers developers and security teams to choose the best components
+    and govern open source usage, ensuring organizations continue to innovate with less risk.
+
+    ## Core Capabilities
+
+    * **Dependency Management**:
+      Generate a precise software bill of materials for all applications
+      to identify open source components and gather contextual direct
+      and transitive dependency information for faster action.
+    * **One-Click Remediation**:
+      Upgrade to the best components and versions based on real-time intelligence.
+      Available in your preferred IDE or a GitHub pull request.
+    * **Automated Open Source Policy Enforcement**:
+      Create custom security, license, and architectural policies based
+      on application type or organization and contextually enforce those policies
+      across every stage of the SDLC.
+    * **Continuous Monitoring**:
+      Real-time notifications of newly discovered defects,
+      as well as vulnerabilities based on component,
+      risk level or application/container affected.
+
+    ## Extras
+
+    * IDE Integrations: Eclipse, IntelliJ, and Visual Studio
+    * Source Control Integrations: GitHub, GitLab, and Bitbucket
+    * Chrome Browser Extension
```

Comment:
not sure why we would mention unsupported software here...

Reply:
These were specifically written for marketing very recently, so I guess it should stay. https://issues.sonatype.org/browse/INT-2881